### PR TITLE
[15.0][MIG] delivery_ups_oca

### PR DESCRIPTION
**Pull request by jaredkipe**

### CODE REVIEW

**Comment by rousseldenis** [comment]
@jaredkipe Could you fix this ?

**Comment by victoralmau** [comment]
Please, cherry-pick https://github.com/OCA/delivery-carrier/pull/578 to commit history

File: delivery_ups_oca/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Copyright 2020 Hunki Enterprises BV
+# Copyright 2021-2022 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Delivery UPS OCA",
+    "summary": "Integrate UPS webservice",
+    "version": "13.0.2.0.0",
```
Comment (chandni299) [inline delivery_ups_oca/__manifest__.py:7]: We can change `"version": "15.0.1.0.0"`

**Comment by chandni299** [comment]
@jaredkipe Can you please look into the last comments/reviews?

**Comment by wytzejan** [comment]
@jaredkipe In case you continue working on this one I'd like to inform you about this: regarding UPS documentation https://www.ups.com/upsdeveloperkit/manageaccesskeys?loc=en_US it seems that OAuth is required now. I think this is good to know.

@HviorForgeFlow also made this comment in the pull request for v14: https://github.com/OCA/delivery-carrier/pull/639

**Comment by rousseldenis** [comment]
@jaredkipe Do you think you'll have time to fix this ?

**Pull request closed without merging**